File: 백준/Silver/1929. 소수 구하기/소수 구하기.c
#include <stdio.h>
#include <math.h>
#include <stdbool.h>

void getPrime(int n, int m);

int main(void) {
	int n, m;
	scanf("%d %d", &n, &m);
	getPrime(n, m);

	return 0;
}

void getPrime(int n, int m) {
	if (n < 2) n = 2;
	for (int i = n; i < m + 1; i++) {
		bool prime = true;
		for (int j = 2; j < (int)sqrt(i) + 1; j++) {
			if (i % j == 0) {
				prime = false;
				break;
			}
		}
		if (prime == true) {
			printf("%d\n", i);
		}
	}
}

File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
import math

def getPrime(n, m):
    if(n < 2): n = 2
    for i in range(n, m + 1):
        prime = True
        for j in range(2, int(math.sqrt(i)) + 1):
            if(i % j == 0):
                prime = False
                break
        if(prime == True):
            print(i)
            
n, m = map(int, input().split())
getPrime(n, m)